obspy/clients/fdsn/client.py

```diff
@@ -1078,7 +1078,10 @@ def _get_bulk_string(self, bulk, arguments):
                 msg = ("Unrecognized input for 'bulk' argument. Please "
                        "contact developers if you think this is a bug.")
                 raise NotImplementedError(msg)
-        return bulk.encode('ASCII')
+        if PY2:
+            return bulk
+        else:
+            return bulk.encode('ASCII')
```

Comment:
@megies For Python 3 I had to explicitly encode the bulk data string as ASCII (to be `<class 'bytes'>`). For Python 2 the bulk request "string" is of type `future.types.newstr.newstr` and worked as is.

Do you know of a better way to handle this?